regression-test/suites/datatype_p0/nested_types/query/array_functions/test_array_functions_array_with_const.groovy

```diff
@@ -31,7 +32,16 @@ suite("test_array_functions_array_with_const", "p0") {
                 SELECT array_with_constant(-231.37104, -138); 
                 """
     } catch (Exception ex) {
-        assertTrue(ex.getMessage().contains("Array size can not be negative in function:array_with_constant"))
+        assertTrue(ex.getMessage().contains("Array size should in range(0, 1000000) in function"))
+    }
+
+    // -- {server for large array}
+    try {
```

Comment:
don't use `try-catch` to do sql failure test. it doesn't work at all. if a sql didn't fail, the test will success. it's not as expected. please use `test{ sql exception }` instead